1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,10 @@
+import fizzbuzz from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
-
-
+  it("should accept from 1 to 100", () => {
+    expect(fizzbuzz(3, 15)).toBeTruthy();
+  });
+  it("should not accept numbers less than 1", () => {});
```

Comment:
Hi 👋 - I sometimes like list out test cases in advance too. 

Have you tried `it.todo`. I like how the output shows which tests are remaining in the "todo" phase

Reply:
Wasn't aware of this. Thanks.

Reply:
Great Daryl, I didn't know about that @daryl-c . Thanks!